lib/class-wp-rest-widget-types-controller.php

```diff
@@ -398,6 +401,11 @@ public function get_item_schema() {
 					'default'     => '',
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'is_multi'                    => array(
+					'description' => __( 'Whether the widget supports multiple instances', 'gutenberg' ),
+					'type'        => 'boolean',
+					'context'     => array( 'view', 'edit', 'embed' ),
```

Comment:
Should get a `readonly`.

Reply:
👍 be026196c4fd4cbf1c751828c1f84404d1209bff